zkml/src/layers/provable/error.rs

```diff
@@ -0,0 +1,33 @@
+//! Contains the Error enum for things to do with an implementor of [`super::ProvableOp`].
+
+#[derive(thiserror::Error, Debug)]
+pub enum ProvableOpError {
```

Comment:
I understood we decided to remove custom errors? Why are we reintroducing them here?